zetaclient/context/app.go

```diff
@@ -39,9 +51,10 @@ func New(cfg config.Config, logger zerolog.Logger) *AppContext {
 
 		chainRegistry: NewChainRegistry(),
 
-		crosschainFlags:  observertypes.CrosschainFlags{},
-		currentTssPubKey: "",
-		keygen:           observertypes.Keygen{},
+		crosschainFlags:     observertypes.CrosschainFlags{},
+		currentTssPubKey:    "",
+		keygen:              observertypes.Keygen{},
+		relayerKeyPasswords: make(map[chains.Network]string),
```

Comment:
No need for this map

Reply:
same.

Reply:
And we shouldn't use a struct to index, we should use a id or string

Reply:
It's an `int32`. I changed it to string to not relay on protobuf implementation of the enum. https://github.com/zeta-chain/node/pull/2673/commits/ad8773ffd130522be972d94e699a74c76144f7d4